pkg/liquidity-mining/contracts/gauges/StakelessGaugeCheckpointer.sol

```diff
@@ -263,6 +271,67 @@ contract StakelessGaugeCheckpointer is IStakelessGaugeCheckpointer, ReentrancyGu
         }
     }
 
+    function _getSingleBridgeCost(IStakelessGauge gauge) internal view returns (uint256) {
+        // Some versions of the stakeless gauges did not implement this interface, so we need to try / catch the call.
+        // In case the interface is not present, the cost is 0.
+        // Malicious contracts are ruled out at this stage: gauges shall be validated in external functions before
+        // reaching this point.
+        try gauge.getTotalBridgeCost() returns (uint256 cost) {
+            return cost;
+        } catch {
+            return 0;
+        }
+    }
+
+    function _getGaugeTypeTotalBridgeCost(string memory gaugeType, uint256 minRelativeWeight)
+        internal
+        view
+        returns (uint256 totalCost)
+    {
+        uint256 currentPeriod = _roundDownBlockTimestamp();
+        uint256 gaugesAmount = _gauges[gaugeType].length();
```

Comment:
`gaugeAmount` is a weird name for a length. It sounds like it refers to the value or weight or something. `numGauges` or `gaugeCount` or something more length-y?